test/unit/setup-steps-test.js

```diff
@@ -43,4 +44,16 @@ describe('Load Translations', function () {
         sandbox.stub(Language, 'loadJsonTranslation').rejects({ code: ERROR_AJAX });
         testLoadTranslations(ERROR_LOADING_TRANSLATIONS + ERROR_AJAX);
     });
+
+    it('should be RTL when RTL translation loads successfully', function () {
+        const rtlLang = 'ar';
+        const rtlModel = Object.assign({ language: rtlLang, intl: {} }, model);
+        rtlModel.attributes.language = rtlLang;
+        sandbox.stub(Utils, 'ajax').callsFake(({ oncomplete }) => {
+            oncomplete({ response: {} });
+        });
+        loadTranslations(rtlModel).then(() => {
+            expect(Language.isRtl(rtlModel.language, rtlModel.intl)).to.be.true;
```

Comment:
If you really want this to return false before the translation is loaded you should test that as well (you can add the `expect.to.be.false` before loading in this test case.